prov/verbs/src/verbs_eq.c

```diff
@@ -439,7 +459,11 @@ fi_ibv_eq_xrc_connected_event(struct fi_ibv_eq *eq,
 	 * ID(s) since  RDMA CM is used for connection setup only */
 	*acked = 1;
 	rdma_ack_cm_event(cma_event);
-	fi_ibv_free_xrc_conn_setup(ep);
```

Comment:
Where did this function move to? Won't the resources be dangling even after a call to fi_close on the endpoint?

Reply:
The close of the EP already had to handle the case of freeing resources that were not previously released; so it will clean them up in fi_ibv_ep_close() -> fi_ibv_ep_xrc_close().